### DISCUSSION TITLE
Is there anything special needed to build?

### DISCUSSION THREAD

**Discussion by SteveDunn** [stryker-mutator/stryker-net#2602] [Q&A]
I just forked and clone the repo.  I get this from Visual Studio/Rider/command line:

```
5>C:\git\thirdparty\stryker-net\src\Stryker.Core\Stryker.Core.UnitTest\Stryker.Core.UnitTest.csproj : error NU1403: Package content hash validation failed for runtime.native.System.Net.Security.4.3.0. The package is different than the last restore.
5>C:\git\thirdparty\stryker-net\src\Stryker.Core\Stryker.Core.UnitTest\Stryker.Core.UnitTest.csproj : error NU1403: Package content hash validation failed for System.Threading.Tasks.Parallel.4.3.0. The package is different than the last restore.
```

Any ideas?

TIA, Steve.

**Comment by rouke-broersma** [comment]
Sounds like you have some nuget packages cached on your system that fail the hash validation. I suggest disabling the "offline packages" package source and clearing your package caches. IIRC there were some issues with some nuget versions and hash calculation in the past and if you still have local packages that were at some point restored with those nuget versions this doesn't get fixed automatically. The reason you've probably never had an issue with this is because almost no-one uses package lock files yet for dotnet, but this project does.
